CMakeLists.txt

```diff
@@ -88,16 +88,16 @@ endif (PAASS_USE_DAMM)
 #------------------------------------------------------------------------------
 #Find packages needed for the software.
 #Load additional find_package scripts.
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/Cmake/modules/")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/Cmake/modules/" "$ENV{XIA_PIXIE_SDK}/cmake/modules")
 
 #Find thread library for poll2 and scanLib
 find_package(Threads REQUIRED)
 
 #Find the PLX Library
```

Comment:
Removing these two `find_package` lines will suppress CMake errors when building only the analysis part of PAASS.

Reply:
`find_package(BroadcomAPI)` and `find_package(PixieSDK)`.